Sources/DateRangePicker/ExpandedDateRangePickerController.swift

```diff
@@ -58,8 +58,49 @@ fileprivate class SolidBackgroundView: NSView {
 	}
 }
 
+extension NSAppearance {
+    fileprivate var isDarkMode: Bool {
+        if #available(macOS 10.14, *) {
+            if self.bestMatch(from: [.darkAqua, .aqua]) == .darkAqua {
+                return true
+            } else {
+                return false
+            }
+        } else {
+            return false
+        }
+    }
+}
 
 open class ExpandedDateRangePickerController: NSViewController {
+    /// - Note: Testing seam to verify the color is loadable from all build products.
+    @available(macOS 10.14, *)
+    static var separatorColor: NSColor? {
```

Comment:
This was actually the hardest part. I learned that `swift build` works fine, but `swift test` (trying to access this color) revealed that the asset cannot be loaded outside of Xcode. This somewhat makes sense, given the `.xcassets` file name :)  (I removed the `resources:` setting from the Package manifest afterwards because it has no effect in the 1 case where it produces a warning, i.e. the command line)

As the comments explain, the problematic code path is irrelevant because this library requires Xcode anyway. Command line builds of AppKit packages don't make sense because Swift packages don't support these yet, anyway, AFAIK.

If you want to see the problem of command line usage in isolation, this project demonstrates the issue:
https://github.com/marcomasser/ColorAssetsInPackageManager